config/routes.rb

```diff
@@ -43,6 +41,10 @@
       get :validation_decision
     end
 
+    resource :constraints, only: %i[show edit update] do
+      post :check
```

Comment:
Is it post or patch here?
I have a feeling it could be patch as we are amending a record rather than creating a new resource